src/FileServiceManager/FileService/DAL/Storages/FileSystemStorage.cs

```diff
@@ -0,0 +1,65 @@
+﻿using FileService.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace FileService.DAL
+{
+    public class FileSystemStorage : IStorage
+    {
+        private readonly IConfiguration _configuration;
+        public FileSystemStorage(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public async Task<FileMetadata> DownLoadFileAsync(Guid fileId, int companyId)
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task<IEnumerable<FileMetadata>> GetAllFilesAsync(int companyId)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<bool> RemoveFileAsync(Guid fileId, int companyId)
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task<Guid> UploadFileAsync(IFormFile file, int companyId)
+        {
+            throw new NotImplementedException();
+            //var basePath = Path.Combine(Directory.GetCurrentDirectory() + "\\Files\\");
```

Comment:
do not leave unused code in committed code

Reply:
Fixed